code/game/objects/items/stacks/stack.dm

```diff
@@ -395,7 +405,20 @@
 	var/window_checks = FALSE
 	var/placement_checks = FALSE
 
-/datum/stack_recipe/New(title, result_type, req_amount = 1, res_amount = 1, max_res_amount = 1, time = 0, one_per_turf = FALSE, on_floor = FALSE, window_checks = FALSE, placement_checks = FALSE)
+/datum/stack_recipe/New(
```

Comment:
This proc input was getting hard to read so I changed it to be read downwards. if I need to change it back let me know

Reply:
downwards is fine ~~but please just indent stuff once or twice instead of aligning it to the paren~~ I don't know if the maintainers care